airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json

```diff
@@ -5,125 +5,125 @@
+            "type": "object",
+            "title": "Microsoft OAuth2.0",
+            "description": "Microsoft API Credentials for connecting to Bings Ads",
+            "required": ["auth_method", "developer_token"],
             "properties": {
-              "selection_strategy": {
+              "auth_method": {
                 "type": "string",
-                "const": "all"
+                "const": "oauth2.0",
+                "order": 0
+              },
+              "developer_token": {
+                "type": "string",
+                "title": "Developer Token",
+                "description": "Developer token associated with user. See more info <a href=\"https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token\"> in the docs</a>.",
+                "airbyte_secret": true,
+                "order": 1
+              },
+              "client_id": {
+                "type": "string",
+                "title": "Client ID",
+                "description": "The Client ID of your Microsoft Advertising developer application.",
+                "airbyte_secret": true,
+                "order": 2
+              },
+              "client_secret": {
+                "type": "string",
+                "title": "Client Secret",
+                "description": "The Client Secret of your Microsoft Advertising developer application.",
+                "airbyte_secret": true,
+                "order": 3
+              },
+              "refresh_token": {
+                "type": "string",
+                "title": "Refresh Token",
+                "description": "Refresh Token to renew the expired Access Token.",
+                "airbyte_secret": true,
+                "order": 4
               }
             }
           },
           {
-            "title": "Specific Accounts",
-            "additionalProperties": false,
-            "description": "Fetch data for subset of account IDs.",
-            "required": ["ids", "selection_strategy"],
+            "type": "object",
+            "title": "Private OAuth2.0 Client",
+            "description": "Microsoft Private Client API Credentials for connecting to Bings Ads",
+            "required": [
+              "auth_method",
+              "developer_token",
+              "client_id",
+              "client_secret",
+              "refresh_token"
+            ],
             "properties": {
-              "selection_strategy": {
+              "auth_method": {
+                "type": "string",
+                "const": "private_client",
+                "order": 0
+              },
+              "developer_token": {
+                "type": "string",
+                "title": "Developer Token",
+                "description": "Developer token associated with user. See more info <a href=\"https://docs.microsoft.com/en-us/advertising/guides/get-started?view=bingads-13#get-developer-token\"> in the docs</a>.",
```

Comment:
Can you please confirm that we do render HTML properly in the UI? I don't know if we sanitize those strings or not

Reply:
<img width="680" alt="Screenshot 2022-05-21 at 22 18 06" src="https://user-images.githubusercontent.com/22987674/169666237-d8639a8f-0aa0-4abd-bbd5-4595e75fff86.png">

It shown something like in this example. The connector on the screenshot is different, but the concept is the same.